Packs/Twitter/Integrations/Twitterv2/Twitterv2.py

```diff
@@ -0,0 +1,360 @@
+        """ Gets tweets according to the query.
+            Args:
+                query: str - The query from the user.
+                start_time: str - Start date from which the Tweets will be provided.
+                end_time: str - The most recent date to which the Tweets will be provided.
+                limit: int - Maximum number of results to return.
+                next_token: str - A value that encodes the next 'page' of results that can be requested.
+
+            Returns:
+                A tuple[dict, dict, Optional[str]] with:
+                List[dict]: raw response.
+                List[dict]: context data.
+                str: next token.
+        """
+        result: List[dict] = []
+        query_params = {'query': ''.join(f'"{item}"' for item in query),
+                        'tweet.fields': 'id,text,attachments,author_id,conversation_id,created_at,public_metrics',
+                        'expansions': 'attachments.media_keys,author_id',
+                        'media.fields': 'media_key,type,url,public_metrics,alt_text',
+                        'user.fields': 'id,name,username,created_at,description,verified',
+                        'max_results': f'{limit}',
+                        'start_time': start_time,
+                        'end_time': end_time,
+                        'next_token': next_token}
+        try:
+            response = self._http_request(method="GET", url_suffix='/tweets/search/recent',
+                                          headers=self._headers, params=query_params,
+                                          ok_codes=[200])
+            result, next_token = create_context_data_search_tweets(response)
+        except Exception as e:
+            raise e
+        return response, result, next_token
+
+    def twitter_user_get(self, users_names: list[str], return_pinned_tweets: str) -> tuple[dict, list[dict]]:
+        """ Gets users according to the provided user names.
+            Args:
+                user_name: list[str] - List of users names.
+                return_pinned_tweets: str - Indicates whether to return user's pinned Tweets.
+                limit: int - Maximum number of results to return.
+
+            Returns:
+                A tuple[dict, dict, Optional[str]] with:
+                dict: raw response.
+                dict: context data.
+        """
+        result = []
+        query_params = {'usernames': ','.join(f'{item}' for item in users_names),
+                        'user.fields': 'created_at,description,entities,id,location,name,pinned_tweet_id,profile_image_url'
+                        ',protected,public_metrics,url,username,verified,withheld'}
+        if return_pinned_tweets == 'true':
+            query_params['expansions'] = 'pinned_tweet_id'
+            query_params['tweet.fields'] = 'id,text,attachments,conversation_id,created_at,public_metrics'
+        try:
```

Comment:
Why do we need the try-except here?
If the request will fail, it will throw an error, no?